src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionForm.tsx

```diff
@@ -80,7 +85,7 @@ const ProposedSolutionForm: React.FC<ProposedSolutionFormProps> = ({
           onSubmit={(e) => {
             e.preventDefault();
             e.stopPropagation();
-            handleSubmit(onAdd)(e);
+            handleSubmit(handleAdd)(e);
```

Comment:
The submit button on this page should say 'Save' when you're editing instead of 'Add'